radio/src/targets/taranis/hal.h

```diff
@@ -2510,7 +2512,7 @@
   #define HAPTIC_GPIO_PIN               GPIO_Pin_12
 #endif
 
-#if defined(RADIO_BOXER)
+#if defined(FLYSKY_GIMBAL)
```

Comment:
This locks the Flysky gimbals to UART4. IMHO, the code would read clearer if here the radios would be listed that use UART4 for Flysky digital hall gimbal connection.

Reply:
Plus the HAL is target specific, so really shouldn't use feature flags unless nested.

Reply:
I see, so that is why it use radio definition instead.  Anyway, let me make it works first.

Reply:
However, for implementation seems both F2 and F4 targets can make use UART4 for the choice of UART, will see if everything works fine first.

Reply:
Even if presently UART4 will be used on all targets capable of accommodating Flysky digital gimbals, this might not be the case for all future radios. By listing here a define by radios, instead of a feature based define, would make it easier in the future to locate the snippet needing augmentation.